test/framework/test_block_generator.cpp

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <chrono>
+#include "framework/test_block_generator.hpp"
+#include "model/model_hash_provider_impl.hpp"
+#include "model/commands/add_peer.hpp"
+#include "model/commands/create_domain.hpp"
+#include "model/commands/create_domain.hpp"
+#include "model/commands/create_asset.hpp"
+#include "model/commands/create_account.hpp"
+#include "model/commands/set_permissions.hpp"
+#include "common/types.hpp"
+
+using namespace iroha;
+using namespace iroha::model;
+
+namespace framework {
```

Comment:
I think, we will remove this file after https://github.com/hyperledger/iroha/pull/503